scripts/globals/mobskills/impulse_drive.lua

```diff
@@ -0,0 +1,39 @@
+---------------------------------------------
+-- Impulse Drive
+-- Delivers a two-hit attack
+-- Type: Physical
+-- Range: Melee
+---------------------------------------------
+require("scripts/globals/mobskills")
+require("scripts/settings/main")
```

Comment:
It doesn't look like any settings are used in this file, so this require isn't needed.